.travis.yml

```diff
@@ -2,31 +2,22 @@ language: ruby
 sudo: false
 
 rvm:
-  - 2.3.1
-  - 2.4.0-preview1
+  - 2.4.0
+  - 2.5.0
+  - 2.6.1
 
 gemfile:
-  - Gemfile
-  - Gemfile_ar32
-  - Gemfile_ar40
+  - Gemfile_ar41
   - Gemfile_ar50
   - Gemfile_ar51
   - Gemfile_ar_master
 
 matrix:
   fast_finish: true
   allow_failures:
-    - rvm: 2.4.0-preview1
-      gemfile: Gemfile
-    - rvm: 2.4.0-preview1
-      gemfile: Gemfile_ar32
-    - rvm: 2.4.0-preview1
-      gemfile: Gemfile_ar40
-    - rvm: 2.4.0-preview1
-      gemfile: Gemfile_ar50
-    - rvm: 2.4.0-preview1
-      gemfile: Gemfile_ar51
-    - rvm: 2.4.0-preview1
+    - rvm: 2.4.0
       gemfile: Gemfile_ar_master
-    - rvm: 2.3.1
+    - rvm: 2.5.0
+      gemfile: Gemfile_ar_master
+    - rvm: 2.6.1
```

Comment:
wanna do 2.6.2 instead, since it’s out? https://www.ruby-lang.org/en/news/2019/03/13/ruby-2-6-2-released/

Reply:
updated to newest version of ruby on each minor version